build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfoes.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: ipblocksinfoes.crd.nsx.vmware.com
+spec:
+  group: crd.nsx.vmware.com
+  names:
+    kind: IPBlocksInfo
+    listKind: IPBlocksInfoList
+    plural: ipblocksinfoes
+    singular: ipblocksinfo
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: IPBlocksInfo is the Schema for the ipblocksinfo API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          externalIPBlocks:
+            items:
+              properties:
+                cidr:
+                  description: A contiguous IP address space represented by network
+                    address and prefix length.
+                  type: string
+                visibility:
+                  description: |-
+                    Visibility specifies the visibility or scope of the IPBlock and expected
+                    consumption of the IPBlock with same scope. Can be External, Private or PrivateTGW.
+                  enum:
+                  - External
+                  - Private
+                  - PrivateTGW
+                  type: string
+              required:
+              - cidr
+              - visibility
+              type: object
+            type: array
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          privateTGWIPBlocks:
```

Comment:
I think this would be useful to include a description indicating the visibility, as well as the _project_ this CRD is associated with. Maybe indicate that it's working with the project of the `default` `VPCNetworkConfiguration`?

Reply:
I added some description. Please take another look if that makes sense, thanks.